README.md

```diff
@@ -747,6 +747,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Musab Habeeb](https://github.com/Musab1258)
   - [Muthu Annamalai V.](https://github.com/muthuannamalai12)
   - [Maryam Farshbafi](https://github.com/MaryamFarshbafi)
+  - [Mujahid Ali](https://github.com/MujahidAli954)
```

Comment:
Please add your name in line 745, after Muiruriscode.
Let us know if you need any help with that! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable